ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_creation.py

```diff
@@ -27,6 +27,22 @@ def _random_cp_data(draw):
     return shape, rank, dtype[0], full, orthogonal, seed, normalise_factors
 
 
+@st.composite
+def _random_tt_data(draw):
+    shape = draw(
+        st.lists(helpers.ints(min_value=1, max_value=5), min_size=2, max_size=4)
+    )
+    rank = draw(helpers.ints(min_value=1, max_value=len(shape)))
+    dtype = draw(
+        helpers.get_dtypes("float", full=False).filter(
+            lambda x: x not in ["bfloat16", "float16"]
```

Comment:
I think there is an internal error related to these dtypes. I remember Ved mentioning this in a sync sometime back.

Reply:
I'm not aware of the internal error related to these dtypes but we shouldn't be adding this condition in the tests. It will automatically be fixed once the internal error is fixed. We shouldn't worry about this if this is the only reason why the testa are failing :)